src/main/scala/org/ua/ilm/Main.scala

```diff
@@ -26,18 +26,20 @@ object Main {
       val idChannelsFile = new File(idChannels.getOrElse(config.paramConfig("channelsId")))
 
       val idChannelsReader = new FileReader(idChannelsFile).readFileWithSeparator()
-      val youTubeCrawlerChannels = new CrawlerChannels(key.get, idChannelsReader, partChannels)
-      val youTubeRequestStr = youTubeCrawlerChannels.channelListRequest()
+      val arrIdChannels = idChannelsReader.split(',')
+
+      val crawler = new ParallelCrawlerChannels(key.get, arrIdChannels, partChannels)
+      val channelsOfJson = crawler.getChannelsOfJson
 
       val writer = new FileWriter(pathOutFile)
-      val result = Try(writer.writeFile(youTubeRequestStr.toPrettyString))
+      val result = Try(writer.writeFile(channelsOfJson))
       result match {
         case Success(value) => println("It`s OK!")
         case Failure(exception: FileNotFoundException) => println("No such file: " + exception)
         case Failure(ex) => println(ex)
       }
-      val list = for {y <- youTubeRequestStr.getItems.toArray} yield YoutubeChannelParser.parse(y.toString)
-      list.foreach(println)
+      val listChannels = new ParallelChannelParser(2).getChannels(channelsOfJson)
```

Comment:
listChannels means print all of channels in a list form (action). In your case "channelsList" (entity). English, my friend